### PR TITLE
Frontend/license search tags

### LINKED ISSUE #312: Licensee list search tags UI
The staff UI for the licensee list has some design updates to include some tags the describe the current search criteria:

Image
Image

### PR DESCRIPTION
**Pull request by jsandoval81**
### Requirements List
- _None_

### Description List
- Add a close (x) icon component
- Add search fields display tag to license list
- Allow the previous search values to be preserved when re-visiting the search form
- Updated the size of the search button to match the updated designs

### Testing List
- `yarn test:unit:all` should run without errors or warnings
- `yarn serve` should run without errors or warnings
- `yarn build` should run without errors or warnings
- Code review
- After searching licensees:
    - If no search terms were requested, display tag does not appear in list view
    - If search terms were requested, display tag should display above table next to search button
    - Search tag should show the requested search values
    - Clicking the X on the tag should take the user back to the search form with previous values cleared out
    - Clicking the Search button should take the user back to the search form preserving the previous values

Closes #312

### CODE REVIEW

File: webroot/src/components/Licensee/LicenseeList/LicenseeList.vue
Comment (ChiefStief) [inline webroot/src/components/Licensee/LicenseeList/LicenseeList.vue:24]: I cant tell immediately why but there are spaces being added before the commas between term

**Review by ChiefStief — CHANGES_REQUESTED** [review]
Looks good. I think I would expect tags to be individual for each search term and each handle toggling off individually / not exiting the search screen when done. If we are doing it this way I'm not sure tags is the most intuitive way to get the idea across.

 That being said this is perfectly fine for now and works well other than the space being added before commas

File: webroot/src/components/Licensee/LicenseeList/LicenseeList.vue
Comment (ChiefStief) [reply webroot/src/components/Licensee/LicenseeList/LicenseeList.vue:24]: It looks like that is a function of having all the terms listed here on separate lines, the white space is bleeding in.

I might make a searchText computed concatenating all these.

**Review by ChiefStief — APPROVED** [review]

**Comment by jsandoval81** [comment]
@jlkravitz This is ready for your review.

**Review by jlkravitz — APPROVED** [review]
@isabeleliassen Good to merge once #352 is approved and merged.

**Review by isabeleliassen — APPROVED** [review]

**Pull request merged**
